From f5ff81d14fdd34f770120a057804d6067fd5035b Mon Sep 17 00:00:00 2001
From: Paul Luse <paul.e.luse@intel.com>
Date: Thu, 24 Apr 2025 09:48:56 -0700
Subject: [PATCH 34/39] Update README.md

Needed to remove the word "test" as part of testing updated CI workflow.

Signed-off-by: Paul Luse <paul.e.luse@intel.com>
---
 README.md | 1 -
 1 file changed, 1 deletion(-)

diff --git a/README.md b/README.md
index 426e3512..029e0ee2 100644
--- a/README.md
+++ b/README.md
@@ -1,4 +1,3 @@
-test
 **mdadm** is a utility used to create and manage **software RAID** devices implemented through
 **Multiple devices driver (MD)** in kernel. It supports following RAID metadata formats:
 
-- 
2.41.0

